.github/workflows/publish.yaml

```diff
@@ -9,7 +9,7 @@ on:  # NOLINT
 jobs:
   build-n-publish:
     name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
if we use `ubuntu-latest` here, should we include it in the matrix in the test workflow? or should we just specify the latest version here?

Reply:
[2c7936e](https://github.com/sscpac/statick/pull/444/commits/2c7936e67e0215723be6406100da6f3b2c8bc9ea)

Reply:
it looks like you only changed the `publish-docker-image` workflow, not this one.